docs/database/oracledb.md

```diff
@@ -0,0 +1,116 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Oracle Database
+--->
+
+# Semantic conventions for Oracle Database
+
+**Status**: [Release Candidate][DocumentStatus]
```

Comment:
I think we should start with Development. We plan to declare stability for ones marked with RC in ~ a month, but we need to let Oracle bake for a bit.

For example, we'd need prototypes/instrumentations in 3 languages. 

```suggestion
**Status**: [Development][DocumentStatus]
```